src/bournemouth/resources.py

```
@@ -282,7 +225,9 @@ async def handle_chat(
         if self._send_lock is None:
             raise RuntimeError("on_connect must be called before handle_chat")
         history = build_chat_history(payload.message, payload.history)
-        api_key = await self._get_api_key()
+        if self._user is None:
+            raise RuntimeError("on_connect must be called before handle_chat")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)